src/themes/teams/components/Text/textStyles.ts

```diff
@@ -1,30 +1,15 @@
 import * as _ from 'lodash'
 
 import { ComponentStyleFunctionParam, ICSSInJSStyle } from '../../../types'
-import { truncateStyle } from '../../../../styles/customCSS'
-import { TextVariables } from './textVariables'
+import { TeamsTextVariables } from './textVariables'
 import { TextProps } from '../../../../components/Text/Text'
 
 export default {
   root: ({
-    props: {
-      atMention,
-      color,
-      disabled,
-      error,
-      size,
-      weight,
-      important,
-      success,
-      timestamp,
-      truncated,
-      temporary,
-    },
+    props: { atMention, color, disabled, error, important, success, timestamp },
     variables: v,
-  }: ComponentStyleFunctionParam<TextProps, TextVariables>): ICSSInJSStyle => {
+  }: ComponentStyleFunctionParam<TextProps, TeamsTextVariables>): ICSSInJSStyle => {
```

Comment:
One question, just so that I understand better. If this theme is designed so that it would be merged with the base team, shouldn't the variables here be TextVariables? Or we are designing this as it could also be a standalone theme? (One scenario would be, let's say, I want to use some additional styling using some variable from the base theme, because I know that this theme will always be merged on top of that one..) Not sure if this is valid, but I just want to understand how we are going to use them, because if we depend on that and do not merge it with base, then we will have some failings..